src/js/api/index.ts

```diff
@@ -47,8 +47,19 @@ const generateErrorObject = (errorResponse: any) => {
   };
 };
 
-const connectToApi = (path: string, options?: RequestInit): ApiPromise<ApiEntityResponse | ApiPreviewResponse> =>
-  fetch(`${host}${path}`, Object.assign({}, defaultOptions, options))
+// TODO: indicate whether the server returns unauthorized and react to it in sagas
```

Comment:
It might be appropriate to document that the `Authorization` header could get overwritten